api/coda.py

```diff
@@ -2,49 +2,45 @@
 
 from datetime import datetime, timedelta
 import os
-from textwrap import dedent
-from typing import cast, get_args, Optional, Literal
+from typing import cast, get_args, Optional, TYPE_CHECKING
 
 from codaio import Coda, Document, Row
 import pandas as pd
 from structlog import get_logger
 
 
 from api.utilities.coda_utils import (
+    make_updated_cells,
     parse_question_row,
     QuestionRow,
+    QuestionStatus,
     DEFAULT_DATE,
-    make_updated_cells,
 )
 from utilities import is_in_testing_mode
 from utilities.discordutils import DiscordUser
-from utilities.utilities import get_user_handle
+from utilities.serviceutils import ServiceMessage
+from utilities.utilities import fuzzy_contains, get_user_handle, shuffle_df
+
+if TYPE_CHECKING:
+    from utilities.question_query_utils import (
+        QuestionQuery,
+    )
 
-log = get_logger()
 
-QuestionStatus = Literal[
-    "Bulletpoint sketch",
-    "Duplicate",
-    "In progress",
-    "In review",
-    "Live on site",
-    "Marked for deletion",
-    "Not started",
-    "Uncategorized",
-    "Withdrawn",
-]
+log = get_logger()
 
 
 class CodaAPI:
-    """Gathers everything for interacting with coda"""
+    """Gathers everything for interacting with
+    [coda](https://coda.io/d/AI-Safety-Info_dfau7sl2hmG/Get-involved_susRF#_lufSr)."""
 
     # Singleton instance
     __instance: Optional[CodaAPI] = None
 
     # Constants
     CODA_API_TOKEN = os.environ["CODA_API_TOKEN"]
     DOC_ID = (
-        "ah62XEPvpG" if os.getenv("ENVIRONMENT_TYPE") == "development" else "fau7sl2hmG"
+        "bmMz5rbOHi" if os.getenv("ENVIRONMENT_TYPE") == "development" else "fau7sl2hmG"
```

Comment:
For context: I wanted test coda to be up to date with prod and making a new test coda by copying current prod was more straightforward than figuring out how to implement one-way sync, so that's what @plexish did.
IMO we can just delete the old test once this is merged.